llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9590,7 +9532,75 @@ void VPWidenMemoryInstructionRecipe::execute(VPTransformState &State) {
         NewLI = Builder.CreateVectorReverse(NewLI, "reverse");
     }
 
-    State.set(getVPSingleValue(), NewLI, Part);
+    State.set(this, NewLI, Part);
+  }
+}
+
+void VPWidenStoreRecipe::execute(VPTransformState &State) {
+  VPValue *StoredValue = getStoredValue();
+
+  const Align Alignment = getLoadStoreAlignment(&Ingredient);
+  bool CreateScatter = !isConsecutive();
```

Comment:
```suggestion
  bool IsConsecutive = isConsecutive();
```

Reply:
Kept as is for now, as `CreateScatter` seems slightly more descriptive w.r.t. how it is used